source/helpers/get-deinit-handler.ts

```diff
@@ -0,0 +1,15 @@
+export default function getDeinitHandler(deinit: Deinit): VoidFunction {
+	if (deinit instanceof MutationObserver || deinit instanceof ResizeObserver || deinit instanceof IntersectionObserver) {
+		return deinit.disconnect;
```

Comment:
```suggestion
		return () => deinit.disconnect();

Reply:
I see, I had this bug in Firefox too but only with `controller.abort`.